tests/test_run_scripts/integration_tests_aix.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Init
+SCRIPT_PATH=$(pwd)
+BASENAME_CMD="basename ${SCRIPT_PATH}"
+SCRIPT_BASE_PATH=`eval ${BASENAME_CMD}`
+
+if [ $SCRIPT_BASE_PATH = "test_run_scripts" ]; then
+  cd ../../
+fi
+
+export pe_dist_dir="http://pe-releases.puppetlabs.lan/2015.3.3"
+export GEM_SOURCE=http://rubygems.delivery.puppetlabs.net
+export BEAKER_VERSION=2.39.0
```

Comment:
actually this line is not needed, already defined in Gemfile, will delete in the next push